### ISSUE TITLE
Any ideas or thoughts about mapping printers

### ISSUE THREAD

**Issue by Busbar2297** [m1k1o/neko#368]
I think the next logical step is mapping endusers printers inside the app, any thoughts how it can be done or implemented in code?

**Comment by Busbar2297** [comment]
I guess if this will be used as workplace solution i would want to print the page i am viewing on my printers, citrix and Kasim do that, i think they use the universal printer driver + mapping printers via the protocol, rdp can do that, would love to investigate it

**Comment by mbattista** [comment]
Neko is a shared environment. The idea is, that everyone can take control and that you only need a browser for it to work. 

The examples you named need to be installed on your maschine, e.g. citrix workspace. The browser has no way to get your local printer information. And even if there was a way, those printers would need to change every time someone else took control.

So while neko could be used as a workspace solution, dedicated solutions will work better.

**Comment by ne0YT** [comment]
@mbattista it would be possible to send a printer output to the client!
Kasm does this. Just let it create a pdf which will be opened using the client browsers print feature.

**Comment by Busbar2297** [comment]
> [@mbattista](https://github.com/mbattista) it would be possible to send a printer output to the client! Kasm does this. Just let it create a pdf which will be opened using the client browsers print feature.
yup this is what i am rhinking about, but this needs a print redirection protocol i guess
